src/styles/components/button.css

```diff
@@ -92,7 +92,7 @@
 .btn--fancy {
   border-style: solid;
   border-width: 39px 41px;
-  border-image-source: url("../../../static/btn-border.svg");
+  border-image-source: url("./../../static/btn-border.svg");
```

Comment:
GLOBAL NIT: Would it be better as `../../static/btn-border.svg` without `./` prefix?